releasenotes/notes/1.3/binary-arithmetic-gates-6cd2b1c8112febe0.yaml

```
@@ -14,7 +14,7 @@ features_circuits:
   - |
     Added the :class:`.MultiplierGate` for multiplication of two :math:`n`-qubit registers, that is
     :math:`|a\rangle |b\rangle \mapsto |a\rangle |b\rangle |a \cdot b\rangle`.
```

This misses the target register (the gate's docstring has it, I just missed it here)
```suggestion
    :math:`|a\rangle |b\rangle |0\rangle \mapsto |a\rangle |b\rangle |a \cdot b\rangle`.
```